docs/data/roomId.md

```diff
@@ -0,0 +1,5 @@
+# Room Id
+
+A room id is a unique string.
+
+The `RoomId` TypeScript type is the string union of all possible room ids.
```

Comment:
`RoomId` was renamed to `RoomType` in [this commit](https://github.com/normal-devs/klicker-knight/pull/23/commits/45f3381b133ce8834f3f3e9a691b49baa6f51f67). In my mind a `room id` would be unique to a single RoomState instance, but multiple RoomState instances can have the same `RoomType`, which just means that they have the same structure (the same types of properties) but not necessarily the same data.